jvm/core/src/test/scripts/test_graph_runtime.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import tvm
+import json
+from tvm.contrib import graph_runtime
+
+def dump_graph_lib(target_dir):
+    n = 4
```

Comment:
I get that this is math inside of a test script but perhaps we want to avoid using single-letter variable names? We could change to `dim` or something like that.